src/script/components/MessagesList/Message/index.tsx

```diff
@@ -145,21 +141,6 @@ const Message: React.FC<
     handleArrowKeyDown(event);
   };
 
-  // when a new conversation is opened using keyboard(enter), focus on the last message
-  useEffect(() => {
-    if (!messageRef.current) {
-      return;
-    }
-    if (history.state?.eventKey === 'Enter') {
-      handleFocus(totalMessage - 1);
-
-      // reset the eventKey to stop focusing on every new message user send/receive afterwards
-      // last message should be focused only when user enters a new conversation using keyboard(press enter)
-      history.state.eventKey = '';
-      window.history.replaceState(history.state, '', window.location.hash);
-    }
-  }, [totalMessage]);
-
```

Comment:
This logic has been moved to the parent container. The handling of focusing a message is now handled by the message list